pkgs/development/libraries/proj/default.nix

```diff
@@ -16,13 +16,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "proj";
-  version = "9.4.1";
+  version = "9.5.0";
 
   src = fetchFromGitHub {
     owner = "OSGeo";
     repo = "PROJ";
     rev = finalAttrs.version;
-    hash = "sha256-sLlG9NNHST9d0G5hV1tOGpTSv4rbUxERW3kwGC+t1iU=";
+    hash = "sha256-j7Bvv8F9wxrcQlquRa7Pdj5HTUJhwo8Wc1/JbULkUhM=";
   };
 
   patches = [
```

Comment:
already fixed in libcurl?

Reply:
The problem is that anything depending on PROJ would have to add cURL to its `buildInputs`: https://github.com/OSGeo/PROJ/blob/9.5.0/cmake/project-config.cmake.in#L25-L27
With this patch we can avoid putting `curl` in `propagatedBuildInputs`.